src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+
+      <a class="logo" href="#home">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <div class="header-menu">
+        <nav class="header__nav nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link is-active" href="#">Apple</a>
+            </li>
```

Comment:
please add empty spaces between the `<li>` blocks, as it will make code cleaner
<img width="1317" alt="Screenshot 2023-10-07 at 16 03 02" src="https://github.com/mate-academy/layout_moyo-header/assets/64746943/d01cb0da-da01-4abe-b544-4eb1cf602c45">